README.md

```diff
@@ -299,7 +127,7 @@ return RNFS.unlink(path)
 
 ```javascript
 // require the module
-var RNFS = require('react-native-fs');
+var RNFS = require('@taoqf/react-native-fs');
```

Comment:
👎

Reply:
Sorry for that, It has been a long time, and I really do not know if `react-native-fs` would work now or not and I don't have time to do some tests.
And, I could not resolve the conflicts in `android/src/main/java/com/rnfs/RNFSManager.java`.